packages/oui-guide-menu/guide-menu.less

```diff
@@ -0,0 +1,43 @@
+@import './_normalize';
+@import './_variables';
+
+.oui-guide-menu {
+  #oui > .guide-menu-normalize();
+
+  display: block;
+
+
+  &__button {
+    color: @oui-guide-button-font-color;
+    background: none;
+    border: none;
+    padding: 0;
+    font: inherit;
+    cursor: pointer;
+    outline: inherit;
+    white-space: nowrap;
+  }
+
+  &__button-icon {
```

Comment:
Element of element: `block__element-element` ? We should be careful with that.

What do you think of using `oui-guide-button`, `oui-guide-button__icon` and `oui-guide-button__text` ?
Or why not a variant of `oui-button` ? :)

Reply:
I have the feeling `oui-guide-button` is the best solution to keep the notion of block, go for it !